arch/arc/Kconfig

```diff
@@ -94,11 +95,19 @@ config ARC_FIRQ
 	prompt "FIRQ enable"
 	default y
 	help
-	Fast interrupts are supported (FIRQ). If FIRQ enabled, for interrupts
-	with highest priority, status32 and pc will be saved in aux regs,
-	other regs will be saved according to the number of register bank;
-	If FIRQ is disabled, the handle of interrupts with highest priority
-	will be same with other interrupts.
+	  Fast interrupts are supported (FIRQ). If FIRQ enabled, for interrupts
+	  with highest priority, status32 and pc will be saved in aux regs,
+	  other regs will be saved according to the number of register bank;
+	  If FIRQ is disabled, the handle of interrupts with highest priority
+	  will be same with other interrupts.
+
+config ARC_USERSPACE
```

Comment:
You could get rid of this option if you don't need it to 'select' other options.
It only exists on x86, for example, to ensure that certain GDT and TSS options are present.
Having said that, I think you will at least need to select THREAD_STACK_INFO for full userspace support.

Reply:
it makes sensor. Now ARC_USERSPACE is useless, i  will remove it. THREAD_STACK_INFO is enabled by ARC_MPU_OPTION.